src/main/webapp/synchronizer.js

```diff
@@ -25,16 +27,15 @@ function startVideoSyncTimer() {
   }, /* ms= */ TIME_INTERVAL);
 }
 
-/** Stops video sync timer. */
-function stopVideoSyncTimer() {
-  clearInterval(videoSyncTimer);
-}
-
 /**
  * Calls functions that seek transcript, and discussion to {@code currentTime}
  * (number of seconds since video started playing).
  */
```

Comment:
can you update the JSDoc to reflect how the calls are not made if the time has not changed?

Reply:
Done, thanks!